registered-json-schemas/PortalDataset.json

```diff
@@ -1,60 +1,54 @@
 {
-  "additionalProperties": false,
+  "$id": "https://repo-prod.prod.sagebase.org/repo/v1/schema/type/registered/org.synapse.nf-portaldataset",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
   "description": "A slightly more specialized dataset concept intended for the specific scope of the NF Portal; see https://nf.synapse.org/Explore/Datasets.",
   "properties": {
     "accessType": {
-      "description": "Access type for the dataset.",
-      "title": "Access Type",
+      "description": "",
       "enum": [
         "Public Access",
         "Open Access",
         "Controlled Access",
         "Private Access"
       ],
+      "title": "AccessTypeEnum",
```

Comment:
So one small annoyance with the LinkML JSON schema generator compared with the previous `make` steps is that `titles` aren't as nice.